doc/book/examples/sources/basics/strings.move

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module 0x0::strings {
+    use sui::id::VersionedID;
+    use sui::tx_context::{Self, TxContext};
+
+    // Use this dependency to get a type wrapper for UTF8 Strings
+    use sui::utf8::{Self, String};
```

Comment:
Now that https://github.com/move-language/move/commit/a8383d88fa48f4e1e0e91264cffbbd27136e4732 landed, we can use this and get rid of the Sui-specific variant!

I think it's also helpful to show some examples of string literals--Move source has nonstandard syntax for these and I always forget what they look like.